.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Create a report to help us improve. Write in English, please.
+title: "[BUG] "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for creating a bug report to help us improve!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: bug-reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: Steps to reproduce the behavior. For example, "1. Go to '...', 2. Click on '...', 3. See error"
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain your problem.
+    validations:
+      required: false
+  - type: input
```

Comment:
could you update it so this type is a dropdown with the SO options?

Reply:
Will Windows and Linux be sufficient or do we need to specify the version/distribution?

Reply:
windows, linux and mac, and maube a optional field below so the user can say say the version (win7, 10, 11, or arch, on manjaro, or ubuntu, etc)

Reply:
Yeah, when I commented I forgot about the version. I think the way you did is good for now